### ISSUE TITLE
Suggested key binding for minor mode gives an error

### ISSUE THREAD

**Issue by chookity-pokk** [ieure/hyperspace-el#2]
```
(eval-after-load "hyperspace"
  (define-key hyperspace-minor-mode-map "\C-cs" #'hyperspace))
```
Using the above code, pulled from the README, in my init file gives the error 
```
Wrong number of arguments: (1 . 2), 0
```
I have tried copy and pasting it into my init file and typing it out myself and still get the same error.

**Comment by ieure** [comment]
Fixed in a7ea085baf4a51cac0513cb57216677722938781.

Correct form is:

    (eval-after-load "hyperspace"
      (lambda () (define-key hyperspace-minor-mode-map "\C-cs" #'hyperspace)))

**Comment by chookity-pokk** [comment]
Sweet, thank you!
